### PR TITLE
Changes in documentation and location of template under postgresql folder in .prep

### PR DESCRIPTION
**Pull request by sadigopu**
## Purpose

1. Documentation changes in README - reordering and new instructions added
2. Moved the env variables template across folders in .prep

## Does this introduce a breaking change?

```
[ ] Yes
[ x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[x ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code
xxx
```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

none
```

## What to Check
Verify that the following are valid
* changed docs.

## Other Information
<!-- Add any other helpful information that may be needed here. -->

### CODE REVIEW

File: .prep/initial-mysql/agoncal-application-petstore-ee7/set-env-variables-template.sh [hunk 1]
```diff
@@ -39,4 +35,7 @@ export MYSQL_SERVER_ADMIN_FULL_NAME=${MYSQL_SERVER_ADMIN_LOGIN_NAME}\@${MYSQL_SE
 
 export FTP_HOST=<my ftp host>
 export FTP_USERNAME=<my ftp user name>
-export FTP_PASSWORD=<my ftp password>
\ No newline at end of file
+export FTP_PASSWORD=<my ftp password>
+
+#IPCONFIG
```
Comment (selvasingh) [inline .prep/initial-mysql/agoncal-application-petstore-ee7/set-env-variables-template.sh]: If you would like to have dev box IP address in the script file, then suggest using DEVBOX_IP_ADDRESS.

DEVBOX_IP_ADDRESS will continue to change as the box moves from one network to another, suggest that you call it out

File: .prep/initial-postgresql/agoncal-application-petstore-ee7/set-env-variables-template.sh [hunk 2]
```diff
@@ -30,3 +26,6 @@ export FTP_HOST=<my ftp host>
 export FTP_USERNAME=<my ftp user name>
 export FTP_PASSWORD=<my ftp password>
 
+#IPCONFIG
+export IPCONFIG=52.247.195.77
```
Comment (selvasingh) [inline .prep/initial-postgresql/agoncal-application-petstore-ee7/set-env-variables-template.sh]: Suggest not including specific configuration

File: VMCreation.txt [hunk 3]
```diff
@@ -0,0 +1,63 @@
+#install Azure CLI
```
Comment (selvasingh) [inline VMCreation.txt]: What do you think about using an MD file for this?

Comment (selvasingh) [reply VMCreation.txt]: You probably want to call the MD file as - Create-Linux-Dev-Machine.MD

File: commands.txt [hunk 4]
```diff
@@ -0,0 +1,38 @@
+git clone --recurse-submodules https://github.com/sadigopu/migrate-Java-EE-app-to-azure.git
```
Comment (selvasingh) [inline commands.txt]: Not sure about the purpose of this file. Looks like credentials are crawling in

File: errors.txt [hunk 5]
```diff
@@ -0,0 +1,12 @@
+1. On trying to add a new Distributed cache for hibernate, system threw a WFLYCTL0369 error - Required capabilities are not available : org.wildfly.clustering.infinispan.transport.channel.hibernate Possible registration points
```
Comment (selvasingh) [inline errors.txt]: Not sure about the purpose of this file

File: ftpdetails.txt [hunk 6]
```diff
@@ -0,0 +1,15 @@
+"publishUrl": "ftp://waws-prod-bay-063.ftp.azurewebsites.windows.net/site/wwwroot",
```
Comment (selvasingh) [inline ftpdetails.txt]: Not sure about the purpose of this file. Probably they are accidentally crawling in

File: readyworkshopcommands.txt [hunk 7]
```diff
@@ -0,0 +1,161 @@
+### Step ONE - Clone and Prep
```
Comment (selvasingh) [inline readyworkshopcommands.txt]: Not sure about the purpose of this file

File: .prep/initial-mysql/agoncal-application-petstore-ee7/set-env-variables-template.sh (hunk 1 above)
Comment (sadigopu) [reply .prep/initial-mysql/agoncal-application-petstore-ee7/set-env-variables-template.sh]: Done and checked-in

**Pull request merged**